modules/base/mw.cx.SiteMapper.js

```
@@ -122,12 +128,19 @@ mw.cx.SiteMapper.prototype.getPageUrl = function (language, title, params, hash)
 	if (this.isMobileDomain()) {
 		prefix += '.m';
 	}
-	let base = this.siteTemplates.view;
+	var templates = this.siteTemplates;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)